lib/redmine/menu_manager/wiki_menu_helper.rb

```diff
@@ -50,7 +50,6 @@ def push_wiki_main_menu(menu, main_item, project)
               { controller: '/wiki', action: 'show', id: main_item.slug },
               param: :project_id,
               caption: main_item.title,
-              after: :repository,
```

Comment:
I'd like to have this changed to a 
```
before: :members
```

so that the two settings-menus are immediate siblings:

![image](https://user-images.githubusercontent.com/617519/127838687-646d9a96-f4fa-4690-a894-42c8e7423361.png)

as opposed to having them split apart:

![image](https://user-images.githubusercontent.com/617519/127838918-e26fcf81-ae83-4d36-84a2-6f2025c8133e.png)

Reply:
Whenever you use a `before` or `after`, the items are being reorderd so the latest one added "wins" and is added first. We'd have to sort them after the previous items.

Reply:
Another option would be to add the "Members" back into the "Project settings" menu where it would become a submenu entry. This would be my preferred choice. Do you have an opinion on this @lindenthal ?